components/cluster/command/root.go

```diff
@@ -115,7 +123,8 @@ func init() {
 			teleCommand = getParentNames(cmd)
 
 			if gOpt.NativeSSH {
-				zap.L().Info("Native ssh client will be used",
+				gOpt.SSHType = executor.SSHTypeSystem
+				zap.L().Info("System ssh client will be used",
```

Comment:
It's better to use `fmt.Println("The --native-ssh flag has been deprecated, please use --ssh=system")` to hint users.